### PR TITLE
Azure web app server farm doc changes

### PR DESCRIPTION
**Pull request by ankurch627**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@ankurch627, I made some small changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4684/commits/c2b5130664c8c6d32a34e13398ee5b0cf014f3da)"). 

The dashboard images are broken. When I attempt to view them, I get an "access denied" message.

File: docs/integrations/microsoft-azure/azure-app-service-plan.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+- **Activity logs**. Provide insight into any subscription-level or management group level events that have occurred in the Azure. To learn more, refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
+
+- **Platform Metrics for Azure App Service Plan**. These metrics are available in the [Microsoft.Web/serverfarms](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-serverfarms-metrics) namespace. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/app-service/web-sites-monitor#understand-metrics). App Service plan metrics are available only for plans in Basic, Standard, and Premium tiers.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-plan.md:19]: lets add the screenshot for diagnostic settings if not present

File: docs/integrations/microsoft-azure/azure-app-service-plan.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
 ## Setup
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-plan.md:21]: Configure metric rules is missing

File: docs/integrations/microsoft-azure/azure-app-service-plan.md [hunk 3]
```diff
@@ -14,7 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 For Azure App Service Plan, you can collect the following metrics:
 
-* **Platform Metrics for Azure App Service Plan**. These metrics are available in [Microsoft.Web/serverfarms](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-serverfarms-metrics) namespace. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/app-service/web-sites-monitor#understand-metrics). App Service plan metrics are available only for plans in Basic, Standard, and Premium tiers.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-plan.md:17]: Also need to add a note that this is supported for Dedicated plans (basic, standard, premium etc) only along with the doc link

**Review by himanshu219 — CHANGES_REQUESTED** [review]
minor feedbacks

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 3 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:17]: Done

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 2 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:21]: Done

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 1 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:19]: Done

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 3 above)
Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:17]: Basic, Standard, Premium  and Isolated tiers

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 1 above)
Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:19]: screenshot file is it already pushed ?

File: docs/integrations/microsoft-azure/azure-app-service-plan.md (hunk 3 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-plan.md:17]: Its there on line #19

**Comment by himanshu219** [comment]
@jpipkin1  the screenshot permissions are fixed now.

**Review by jpipkin1 — APPROVED** [review]
@ankurch627 and @himanshu219, thanks for fixing permissions. I can now see the screenshots.

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**